test/s3path.jl

```diff
@@ -159,27 +173,30 @@ end
 
 @testset "$(typeof(ps.root))" begin
     testsets = [
-        test_constructor,
         test_s3_constructors,
         test_registration,
         test_show,
         test_parse,
         test_convert,
         test_components,
-        test_parents,
+        test_indexing,
+        test_iteration,
+        test_s3_parents,
+        test_descendants_and_ascendants,
         test_s3_join,
+        test_splitext,
         test_basename,
         test_filename,
         test_extensions,
         test_isempty,
-        test_s3_norm,
-        # test_real, # real doesn't make sense for S3Paths
+        test_s3_normalize,
+        # test_canonicalize, # real doesn't make sense for S3Paths
```

Comment:
If we're not testing this then we should probably just remove it.

Reply:
I kept it there in case we want to check that `canonicalize(::S3Path)` is a no-op in the future.